requirements.txt

```diff
@@ -1,6 +1,6 @@
-ctranslate2==1.16.2
+ctranslate2==2.10.1
```

Comment:
I think you should not specify a version for ctranslate2 because it is specified by argostranslate, this will avoid problems.

Reply:
I inserted this line before updating `pip`, I'll go check if it works without it on the newer version of pip.

It's possible that this PR may not even be necessary if `pip install` works without modifications to requirements.txt - we just need to tell people in the README to update their `pip` to the latest version before installing from source (virtualenvs, unfortunately, do not install newest pip when creating the environment).